book/website/collaboration/inclusive-events/inclusive-events-tech.md

```diff
@@ -0,0 +1,208 @@
+(cl-inclusive-events-technology)=
+# Technology Considerations for Inclusive Events
+When planning an event that will be attended by people with lived experience, event organizers should carefully consider various technology aspects to ensure inclusivity, accessibility, and a seamless experience for all participants. 
+
+This section focuses on considerations and recommendations around using different technologies to support the inclusion of people with lived experience at both in-person and hybrid (or online) events. 
+
+(cl-inclusive-events-hybrid)=
+## Hybrid as an Accessibility and Inclusion Strategy
+Hybrid events, by default, prioritize accessibility and the inclusivity of attendees with lived experience, making them the ideal choice to engage an audience of individuals with diverse lived experiences. 
+Online sessions can be better supportive of individuals who experience extreme fatigue or find it difficult to attend in-person events, while in-person components can make use of local resources to support engagements and may be preferred for others requiring support.
+
+The integration of both in-person and virtual components ensures that participants can engage in a manner that suits their preferences and circumstances.
+
+The flexibility of hybrid formats recognizes the challenges faced by individuals with varied abilities and mobility constraints, providing a virtual option to overcome any physical barriers presented by in-person only events. 
+This flexibility also accommodates the diverse schedules and responsibilities of all attendees, and acknowledges any additional scheduling challenges that may be faced by people with lived experiences. 
+
+Although planning and hosting hybrid events often requires more resources to be successful, by making your event hybrid you can better prioritize accessibility needs and the inclusion of attendees with lived experience at your event, empowering a wide range of participants to contribute and benefit from your event's content and community.
+
+If you are planning a hybrid event, please review The Turing Way's {ref}`cl-hybrid-collab` and specifically the {ref}`cl-hybrid-collab-guidelines`.
```

Comment:
```suggestion
If you are planning a hybrid event, please read The Turing Way's {ref}`Chapter on Hybrid Collaboration<cl-hybrid-collab> and specifically the {ref}`Subchapter on Guidelines for Hybrid Collaboration<cl-hybrid-collab-guidelines>`.
```

Reply:
Thanks @EKaroune 

I might have to go through and do this in a few other places too as I assumed it rendered the title of the content block you referenced 😅